src/current/v23.2/cutover-replication.md

```diff
@@ -0,0 +1,174 @@
+~~~
+
+Refer to [Using different timestamp formats]({% link {{ page.version.version }}/as-of-system-time.md %}#using-different-timestamp-formats) for more information.
+
+Similarly, to cut over to a specific time in the future:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER standbyapplication COMPLETE REPLICATION TO SYSTEM TIME '+5h';
+~~~
+
+A future cutover will proceed once the replicated data has reached the specified time.
+
+## Step 2. Completing the cutover
+
+1. The completion of the replication is asynchronous; to monitor its progress use:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTER standbyapplication WITH REPLICATION STATUS;
+    ~~~
+    ~~~
+      id |        name         |         data_state          | service_mode | source_tenant_name |                                                source_cluster_uri                                                 | replication_job_id |       replicated_time        |         retained_time         |          cutover_time
+    -----+---------------------+-----------------------------+--------------+--------------------+-------------------------------------------------------------------------------------------------------------------+--------------------+------------------------------+-------------------------------+---------------------------------
+      4  | standbyapplication  | replication pending cutover | none         | application        | postgresql://user:redacted@3ip:26257/?options=-ccluster%3Dsystem&sslmode=verify-full&sslrootcert=redacted         | 903895265809498113 | 2023-09-28 17:41:18.03092+00 | 2023-09-28 16:09:04.327473+00 | 1695922878030920020.0000000000
+    (1 row)
+    ~~~
+
+    Refer to [Physical Cluster Replication Monitoring]({% link {{ page.version.version }}/physical-cluster-replication-monitoring.md %}) for more  detail.
+
+1. Once complete, start the standby's virtual cluster with:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    ALTER VIRTUAL CLUSTER standbyapplication START SERVICE SHARED;
+    ~~~
+
+    ~~~
+      id |        name         |     data_state     | service_mode
+    -----+---------------------+--------------------+---------------
+      1  | system              | ready              | shared
+      2  | template            | ready              | none
+      3  | standbyapplication  | ready              | shared
+    (3 rows)
+    ~~~
+
+1. To make the standby's virtual cluster the default for connection strings. Set the following [cluster setting]({% link {{ page.version.version }}/cluster-settings.md %}):
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SET CLUSTER SETTING server.controller.default_target_cluster='standbyapplication'
+    ~~~
+
+At this point, the primary and standby clusters are entirely independent. You will need to use your own orchestration to direct traffic to the standby.
```

Comment:
This is very vague. I see why, but saying "orchestration" makes you think of Kubernetes, which is not what this is about. I think you mean something like network load balancing. It's not clear to me if the best practice is to nuke and pave the previous primary since we don't support an Active-Active setup, which it seems like this is likely to lead to.

Reply:
I changed to load balancers here. I thought this was vague too — and yes, I guess we can't really go into detail here. I wonder if we can broaden this as we have more customers using it